core/src/aaa/main/game/PlayerInputProcessor.java

```diff
@@ -0,0 +1,85 @@
+package aaa.main.game;
+
+import com.badlogic.gdx.Input;
+import com.badlogic.gdx.InputProcessor;
+import com.badlogic.gdx.physics.box2d.Body;
+
+public class PlayerInputProcessor implements InputProcessor {
+    private float horizontalVelocity;
+    private float verticalVelocity;
+
+    private Body player;
+    public PlayerInputProcessor(Body player) {
+        this.player = player;
+    }
+    @Override
+    public boolean keyDown(int keycode) {
+
+        horizontalVelocity = 0;
+        verticalVelocity = 0;
+
+        if (keycode == Input.Keys.LEFT) {
+            //move left
+            horizontalVelocity = -5;
+        } else if (keycode == Input.Keys.RIGHT) {
+            //move right
+            horizontalVelocity = 5;
+        } else if (keycode == Input.Keys.UP) {
+            //move up
+            verticalVelocity = 5;
+        } else if (keycode == Input.Keys.DOWN) {
+            //move down
+            verticalVelocity = -5;
+        }
+
+        player.setLinearVelocity(horizontalVelocity, verticalVelocity);
+
+        return false;
```

Comment:
not sure what the effect of returning false here is, but i think you should be returning true whenever you take inputs